cognite/neat/graph/loaders/_rdf2asset.py

```diff
@@ -119,29 +138,54 @@ def _load(self, stop_on_exception: bool = False) -> Iterable[AssetWrite | NeatIs
             for identifier, properties in self.graph_store.read(class_.suffix):
                 fields = _process_properties(properties, property_renaming_config)
                 # set data set id and external id
-                fields["data_set_id"] = self.data_set_id
-                fields["external_id"] = identifier
+                fields["dataSetId"] = self.data_set_id
+                fields["externalId"] = identifier
+
+                # check on parent
+                if "parentExternalId" in fields and fields["parentExternalId"] not in processed_instances:
+                    error = loader_issues.InvalidInstanceError(
+                        type_="asset",
+                        identifier=identifier,
+                        reason=(
+                            f"Parent asset {fields['parentExternalId']} has not been processed yet"
```

Comment:
Does this mean that the parent does not exists? Or is there a chance that it can exists somewhere else and thus mis labelled?
```suggestion
                            f"Parent asset {fields['parentExternalId']} does not exist"
```

Reply:
Does not exist is better statement

Reply:
does not exist or failed creation